### PR TITLE
Fix Yourbittorrent, TorrentFunk & Tolock

### PR DESCRIPTION
**Pull request by ShlomiD83**
this fixes Torlock provider not returning any results.

### CODE REVIEW

**Comment by antonsoroko** [comment]
@ShlomiD83 please also do the same for torrentfunk and yourbittorrent (change direct url to backend (which was changed to different domain) to frontend url). your trick works for them as well since all those 3 are the same, as we can see, the only difference is design, but they have same source of torrent files.

**Comment by ShlomiD83** [comment]
@antonsoroko can you please check if I did to correctly?

File: burst/providers/providers.json [hunk 1]
```diff
@@ -2924,7 +2924,7 @@
             "row": "find_once('div', ('class', 'table-responsive'), order=2).find_all('tr', start=2)",
             "seeds": "item(tag='td', order=5)",
             "size": "item(tag='td', order=3)",
-            "torrent": "'https://yt.0c.gay/down/%s.torrent' % item(tag='a', attribute='href', divider=('/', 2))"
+            "torrent": "'https://yourbittorrent.com/?q=QUERYEXTRA/down/%s.torrent' % item(tag='a', attribute='href', divider=('/', 2))"
```
Comment (antonsoroko) [inline burst/providers/providers.json:2927]: remove `/?q=QUERYEXTRA`

**Comment by antonsoroko** [comment]
looks good. now wait for elgatito to merge it.

**Pull request merged**